grpc/metrics/src/main/java/io/helidon/grpc/metrics/GrpcMetrics.java

```diff
@@ -70,6 +66,19 @@
     private static final MetricRegistry APP_REGISTRY =
             RegistryFactory.getInstance().getRegistry(MetricRegistry.Type.APPLICATION);
 
+    static {
+        HelidonFeatures.register("gRPC Server", "Metrics");
+        HelidonFeatures.register("gRPC Client", "Metrics");
+
+        VENDOR_REGISTRY.meter(org.eclipse.microprofile.metrics.Metadata.builder()
+                .withName("grpc.requests.meter")
+                .withDisplayName("Meter for overall gRPC requests")
+                .withDescription("Each gRPC request will mark the meter to see overall throughput")
+                .withType(MetricType.METERED)
+                .withUnit(MetricUnits.NONE)
+                .build());
```

Comment:
I see that this takes the place of the meter that used to be registered in `MetricsSupport`. There is no replacement here for the counter that `MetricsSupport` used to register. Is that intentional?

Reply:
Yes -- meter already has a counter, so having a separate counter is a bit pointless.